userspace/libsinsp/test/suppress.ut.cpp

```diff
@@ -0,0 +1,142 @@
+	                     (uint64_t)pid,                       /* tid */             \
+	                     (uint64_t)pid,                       /* pid */             \
+	                     (uint64_t)pid,                       /* ptid */            \
+	                     "",                                  /* cwd */             \
+	                     (uint64_t)0,                         /* fdlimit */         \
+	                     (uint64_t)0,                         /* pgft_maj */        \
+	                     (uint64_t)0,                         /* pgft_min */        \
+	                     0,                                   /* vm_size */         \
+	                     0,                                   /* vm_rss */          \
+	                     0,                                   /* vm_swap */         \
+	                     comm,                                /* comm */            \
+	                     scap_const_sized_buffer{nullptr, 0}, /* cgroups */         \
+	                     scap_const_sized_buffer{nullptr, 0}, /* env */             \
+	                     0,                                   /* tty */             \
+	                     (uint64_t)0,                         /* vpgid */           \
+	                     0,                                   /* loginuid */        \
+	                     0,                                   /* flags */           \
+	                     (uint64_t)0,                         /* cap_inheritable */ \
+	                     (uint64_t)0,                         /* cap_permitted */   \
+	                     (uint64_t)0,                         /* cap_effective */   \
+	                     (uint64_t)0,                         /* exe_ino */         \
+	                     (uint64_t)0,                         /* exe_ino_ctime */   \
+	                     (uint64_t)0,                         /* exe_ino_mtime */   \
+	                     (uint32_t)0,                         /* uid */             \
+	                     "",                                  /* trusted_exepath */ \
+	                     (uint64_t)0)                         /* pgid */
+
+TEST_F(sinsp_with_test_input, suppress_comm) {
+	//
+	// init (pid 1)
+	//   └── sh (pid 17)
+	//
+
+	add_default_init_thread();
+
+	m_inspector.clear_suppress_events_comm();
+	m_inspector.suppress_events_comm("sh");
+
+	open_inspector();
+
+	uint64_t pid = 17;
+	generate_clone_x_event(pid, INIT_TID, INIT_TID, INIT_TID);
+
+	const char* file_to_run = "/bin/sh";
+
+	add_event_advance_ts(increasing_ts(), pid, PPME_SYSCALL_EXECVE_19_E, 1, file_to_run);
+
+	EXPECT_ANY_THROW(generate_execve_x(pid, file_to_run, "sh"));
+
+	add_event_advance_ts(increasing_ts(), pid, PPME_PROCEXIT_1_E, 0);
+
+	// dummy event to actually delete the thread from the threadtable.
+	add_event_advance_ts(increasing_ts(), INIT_TID, PPME_SYSCALL_RENAME_E, 0);
+
+	auto tinfo = m_inspector.get_thread_ref(pid);
```

Comment:
shouldn't we check the thread is in the table before the proc_exit? Just to be sure that the thread is at least added in the first place